templates/example.gotxt

```diff
@@ -2,7 +2,11 @@
 
 {{- template "doc" .Doc -}}
 
-{{- codeBlock "go" .Code -}}
+{{- codeBlock "go" .Code }}
+
+Output:
+
+{{ codeBlock "output" .Output -}}
```

Comment:
The first argument to `codeBlock` specifies the language to use for syntax highglighting. In this case, there's no syntax highlighting to perform, so `""` would be the expected value. Example (line 5):

https://github.com/princjef/gomarkdoc/blob/ddf65c2925b62cc4a77fa4302006789901c6be4c/templates/doc.gotxt#L1-L9

Reply:
Makes sense. I've seen style guides before where every code fence is expected to have a type attached to it. In this case though, it certainly makes much less sense because it's just arbitrary text.